pyproject.toml

```diff
@@ -10,3 +10,17 @@ dependencies = [
     "numpy>=1.15",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+markers = [
+    "deeprec: tests for deeprec model",
+    "experimental",
+    "flaky: flaky tests that can fail unexpectedly",
+    "gpu: tests running on GPU",
+    "integration: integration tests",
+    "notebooks: tests for notebooks",
+    "sequential",
```

Comment:
@miguelgfierro And this?

Reply:
I think the mark deeprec and sequential can be removed, they only appear in this notebook https://github.com/microsoft/recommenders/blob/main/tests/smoke/recommenders/recommender/test_deeprec_model.py